Homework7/Car.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Homework7
+{
+    public abstract class Car
+    {
+        public string? brand;
+        public string? model;
+        public int year;
+        public Car(string brand, string model, int year)
+        {
+            this.brand = brand;
+            this.model = model;
+            this.year = year;
+        }
+        public abstract int Speed { get; set; }
+        public abstract void EngineStart();
+        public void DisplayCarDetails()
+        {
+            Console.WriteLine($"{brand} {model} {year}");
+        }
+        public void Move()
+        {
+            Console.WriteLine("Wrooom!");
+        }
+    }
+    public class ElectroCar : Car
+    {        
```

Comment:
Install extension "trailing whitespace visualizer" and remove all white spaces 
![image](https://user-images.githubusercontent.com/23722661/201768588-5d7f8b34-7c25-4778-9e1d-229fb2e34729.png)

Reply:
done